apluslms_shepherd/celery_tasks/build/tasks.py

```diff
@@ -74,9 +66,9 @@ def build_repo(pull_result, base_path, course_key, instance_key, build_number):
                     log)
     try:
         if int(build_number) < max(number_list):
-            print(
+            logger.warning(
```

Comment:
I don't think thatall of the slogged strings are warnings. Maybe debug and info should be used for some?

Reply:
I think the information like `"Already have a newer version in the task queue, the task with build number %s aborted.` should be a warning, because this is some event which cause the current task to stop.

However, I think I'll change next log `"Current build numbers: %s"` to info